src/main/java/com/elastic/support/util/RestExec.java

```diff
@@ -140,36 +132,23 @@ protected String getResponseString(HttpResponse response) {
       }
    }
 
-   protected boolean streamResponseToFile(HttpResponse response, String destination) {
+   protected void streamResponseToFile(HttpResponse response, String destination) {
 
       try {
          int status = checkResponseCode(response);
          org.apache.http.HttpEntity entity = response.getEntity();
          InputStream responseStream = entity.getContent();
-
+         FileOutputStream fos = new FileOutputStream(destination);
```

Comment:
You should close the output stream after opening it (using Java 7's [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html)):

```diff
+try (FileOutputStream fos = new FileOutputStream(destination)) {
+    IOUtils.copy(responseStream, fos);
+}
-FileOutputStream fos = new FileOutputStream(destination);
-IOUtils.copy(responseStream, fos);
```